data/sql/updates/pending_db_world/rev_1606102990713316100.sql

```diff
@@ -0,0 +1,21 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1606102990713316100');
+
+-- Imported from TrinityCore by Voxstrasza
+
+-- The Lich King Timed Actionlist
+DELETE FROM `smart_scripts` WHERE (source_type = 9 AND entryorguid = 25462 * 100);
```

Comment:
What about backticks here? Sry for asking because it's approved by 3 experts

Reply:
yeah backticks would be better (though not strictly required), I haven't noticed it

Reply:
Are you talking about the sql or the imported note?